modules/image-loading-optimization/assets/js/preview/index.js

```diff
@@ -0,0 +1,85 @@
+import { onLCP } from './web-vitals';
+
+class WebVitalsChecker extends elementorModules.ViewModule {
```

Comment:
extending `elementorModules.ViewModule` not mandatory here, as we are only using `onInit` from `elementorModules.ViewModule`